src/common/micro/mode_lima_ccn_activation.F90

```diff
@@ -76,6 +77,11 @@ SUBROUTINE LIMA_CCN_ACTIVATION (CST,                                           &
                                 XHYPINTP1, XHYPINTP2, XTMIN, XHYPF32, XPSI3, XAHENG, XAHENG2, XPSI1, &
                                 XLBC, XLBEXC
 USE MODD_NEB_n,           ONLY: LSUBG_COND
+USE MODD_CH_AEROSOL, ONLY: LORILAM
+USE MODD_DUST, ONLY: LDUST
+USE MODD_SALT, ONLY: LSALT
```

Comment:
Work has not been done (yet) for this routine, but we won't be able to keep the LORILAM, LDUST and LSALT variables directly read from the module. They must be received by the interface (encapsulated in a structure or not)